node/requirements.txt

```diff
@@ -0,0 +1,10 @@
+aiohttp==3.7.4.post0
+async-timeout==3.0.1
+attrs==21.2.0
+chardet==4.0.0
+idna==3.3
+multidict==5.2.0
+requirements-parser==0.2.0
+semver==2.13.0
+typing-extensions==3.10.0.2
+yarl==1.7.0
```

Comment:
The _node_ generator doesn't need most of these. If I'm not missing something, it only needs `aiohttp` and, now, `semver`. All the rest look like they belong to other generators.
And we certainly should not pin to exact versions (`==`).